code/datums/diseases/_MobProcs.dm

```diff
@@ -154,8 +154,13 @@
 		return FALSE
 
 	for(var/thing in D.required_organs)
```

Comment:
please for the love of god rename this variable

Reply:
Done!